src/main.rs

```diff
@@ -59,6 +59,44 @@ enum Commands {
         )]
         filter_ids: Vec<String>,
     },
+    Manage {
+        #[clap(
+            short,
+            long,
+            default_value = "18000",
+            help = "The listening port for the management server."
+        )]
+        port: u16,
+        #[clap(
+            short,
+            long,
+            default_value = "18090",
+            help = "The listening port for the admin server."
+        )]
+        admin_port: u16,
+        #[clap(subcommand)]
+        provider: ProviderCommands,
+    },
+}
+
+#[derive(clap::Subcommand)]
+enum ProviderCommands {
+    Agones {
+        #[clap(
+            short,
+            long,
+            default_value = "quilkin",
+            help = "Namespace under which the proxies run."
+        )]
+        config_namespace: String,
+        #[clap(
+            short,
+            long,
+            default_value = "gameservers",
```

Comment:
I recommend we set these both to `default ` for both. That namespace is the default, in a Kubernetes cluster - so if people are testing this out, it probably makes the most sense to point it there, since I expect people will dump everything in there sometimes anyway (or at least on first attempt).

Reply:
That's fine with me, these are just what's in the current xDS implementation.